tests/userProfile.test.js

```diff
@@ -0,0 +1,209 @@
+const axios = require('axios');
+
+const USERS_SERVICE_HOST = "http://peerpreptest.bryanlohxz.com/api/users-service"
+const INCORRECT_PASSWORD_MSG = "The password entered is incorrect.";
+const INVALID_JWT_ERROR_MSG = "JWT is either missing, invalid or has expired.";
+const INVALID_REQUEST_BODY_ERROR_MESSAGE = "Please check your request body.";
+const USER_NOT_FOUND_MSG = "Sorry, the user cannot be found.";
+const USER_WITH_SAME_EMAIL_FOUND_MSG = "Another user with this email already exists."
+
+const TEST_NAME = "John Doe";
+const TEST_EMAIL = "johndoe@example.com";
+const TEST_PWD = "john123";
+
+let token
+
+test('Sign up for new user profile successfully', async () => {
+  const response = await axios.post(`${USERS_SERVICE_HOST}/signup`, {
+    name: TEST_NAME,
+    email: TEST_EMAIL,
+    password: TEST_PWD,
+  });
+  expect(response.status).toBe(200)
+});
+
+test('Sign up for new user profile with existing email', async () => {
+  try {
+    const response = await axios.post(`${USERS_SERVICE_HOST}/signup`, {
+      name: 'Test User With Same Email',
+      email: TEST_EMAIL,
+      password: 'testuser123',
+    });
+  } catch (error) {
+    expect(error.response.status).toBe(400);
+    expect(error.response.data).toBe(USER_WITH_SAME_EMAIL_FOUND_MSG);
+  }
+});
```

Comment:
Have you tested this? I think it wont throw an error but will just return `response` with `response.status !== 200`